bin/setup

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
+echo 'gem: --no-ri --no-rdoc --no-document' > ~/.gemrc
+bundle update --jobs=3
```

Comment:
If we are doing this here, we should stop travis from doing it also i the .travis.yml

Reply:
I might be going blind but I don't see in .travis.yml where we call out to do bundle install, but it definitely is done https://travis-ci.org/ManageIQ/manageiq-schema/jobs/614674353#L413

Reply:
:man_facepalming: nevermind, I was diff'ing the same file and wondering why nothing was different

Reply:
Okay done, only bundle install once now but it looks like it isn't using the bundler cache anymore ;(